Essentials/build.gradle

```
@@ -8,6 +8,10 @@ dependencies {
 
     api 'org.bstats:bstats-bukkit:1.8'
 
+    api 'org.spongepowered:configurate-yaml:4.1.1'
```

```suggestion
    implementation 'org.spongepowered:configurate-yaml:4.1.1'
```

Our internal usage should always be through the wrapper (and is, as of latest commit)